target/linux/realtek/files-5.10/arch/mips/rtl838x/prom.c

```diff
@@ -35,11 +35,16 @@ static struct plat_smp_ops rtl_smp_ops;
 
 static void rtl_init_secondary(void)
 {
+#ifndef CONFIG_CEVT_R4K
 /*
- * MIPS timer interrupt might fire like crazy if not used or initialized
- * properly. Silence it by setting the maximum possible interval.
+ * These devices are low on resources so it is helpful to disable all unneeded
```

Comment:
I think this first sentence about resources can be omitted, as eventually, we _might_ go for a single generic mips kernel for multiple devices, where CEVT_R4K might simply always be enabled.